### ISSUE TITLE
build_targets function confused

### ISSUE THREAD

**Issue by wojiaozhupengfei** [eriklindernoren/PyTorch-YOLOv3#301]
Hi ALL: 
I have a problem with build_targets function.

def build_targets(pred_boxes, pred_cls, target, anchors, ignore_thres):
    # Coordinates
    tx[b, best_n, gj, gi] = gx - gx.floor()
    ty[b, best_n, gj, gi] = gy - gy.floor()
    # Width and height
    tw[b, best_n, gj, gi] = torch.log(gw / anchors[best_n][:, 0] + 1e-16)
    th[b, best_n, gj, gi] = torch.log(gh / anchors[best_n][:, 1] + 1e-16)

for example:
gx - gx.floor() shape is 30, indicate the label of 30 targets, tx[b, best_n, gj, gi] shape is 30,represents assigning 30 label values(gx - gx.floor()) to corresponding location of this tensor.
but b, best_n, gj, gi exist the same coordinate values,because tx[obj_mask].sum() may be 25, indicate that there are 5 coordinate that are the same.So this assignment will cause the latter value to override the previous value.However, the label values are unique.This makes some label values not involved in the calculation.
I am confused about this and hope to get your help.

**Comment by Flova** [comment]
This part of the code has been updated or removed. I, therefore, close this issue now. Feel free to reopen for further discussion.
